lms/djangoapps/course_home_api/progress/v1/serializers.py

```diff
@@ -0,0 +1,22 @@
+# pylint: disable=abstract-method
+"""
+Progress Tab Serializers
+"""
+
+from rest_framework import serializers
+from lms.djangoapps.course_home_api.outline.v1.serializers import CourseBlockSerializer
+
+
+class UserSerializer(serializers.Serializer):
+    username = serializers.CharField()
+    email = serializers.CharField()
+    is_staff = serializers.BooleanField()
```

Comment:
This is all stuff we should be getting from the auth jwt token, right? Let's not add redundant info to our endpoint.

```
    const { administrator, username } = getAuthenticatedUser();
```

(administrator is is_staff) Do we need email?